### PR TITLE
implement global loader

### PR DESCRIPTION
**Pull request by SvetlanaNagovicyna**

### CODE REVIEW

File: src/app/private/pages/home/home.module.ts [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
   declarations: [
     HomeComponent,
   ],
-  imports: [
-    CommonModule,
-  ],
+    imports: [
+        CommonModule,
+        SharedModule,
+    ],
```
Comment (leonxaoss) [inline src/app/private/pages/home/home.module.ts:13]: need to reformat

File: src/app/private/pages/home/home.module.ts [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
 import { CommonModule } from "@angular/common";
+import { SharedModule } from '../../../shared/shared.module';
 
 @NgModule({
   declarations: [
     HomeComponent,
   ],
   imports: [
     CommonModule,
+    SharedModule,
```
Comment (leonxaoss) [inline src/app/private/pages/home/home.module.ts:12]: do you need here SharedModule?

File: src/app/shared/providers/services/user.service.ts [hunk 3]
```diff
@@ -50,6 +52,7 @@ export class UserService {
 
     return this.getUserById(userId).pipe(
       tap((user: User): void => {
+        this.loaderService.hideLoader();
```
Comment (leonxaoss) [inline src/app/shared/providers/services/user.service.ts:55]: maybe better to hide loader in resolver

File: src/app/private/pages/home/home.module.ts (hunk 2 above)
Comment (SvetlanaNagovicyna) [reply src/app/private/pages/home/home.module.ts:12]: No)

**Pull request merged**